src/toolboxes/background/Canvas.tsx

```diff
@@ -17,11 +17,14 @@ interface Props extends BaseProps {
 export class BackgroundCanvasClass extends Component<Props> {
   private baseCanvas: BaseCanvas;
 
-  private image: HTMLImageElement | HTMLCanvasElement;
+  private image: HTMLImageElement | ImageBitmap;
```

Comment:
Is there anytime will pass a HTMLImageElement at this level? Can we add a comment here documenting why why have two potential types?

Reply:
We don't pass a HTMLImageElement in, that's the old method for autoloading zebrafish-heart.jpg - the background canvas creates a `HTMLImageElement` in `loadImage` and assigns it to `this.image` which then gets passed into `drawImageOnCanvas`. We'll be deprecating it once we can load non-tiff images via `upload3DImage` (which we should probably rename to just `uploadImage`), then we can autoload zebrafish-heart.jpg that way and pass it into background canvas as an `ImageBitmap` instead of background canvas having to load it itself.

Reply:
Perfect